ports/libigl/vcpkg.json

```
@@ -21,12 +21,20 @@
     "cgal": {
       "description": "Build with cgal",
       "dependencies": [
-        "cgal"
+        "cgal",
+        {
+          "name": "cgal",
+          "default-features": false
+        }
       ]
     },
     "embree": {
       "description": "Build with embree",
       "dependencies": [
+        {
+          "name": "embree",
```

```suggestion
          "name": "embree3",
```
Is it repeated with the `embree3` below.

---

fixed with https://github.com/microsoft/vcpkg/pull/27963/commits/76a4b123ca3db34752c2a7a3320e2ce172fc2f1e